benchmarks/benchmarks/optimize_linprog.py

```diff
@@ -1,21 +1,17 @@
 """
 Benchmarks for Linear Programming
 """
-from __future__ import division, print_function, absolute_import
 
 # Import testing parameters
-try:
-    from scipy.optimize import linprog, OptimizeWarning
-    from scipy.linalg import toeplitz
-    from scipy.optimize.tests.test_linprog import lpgen_2d, magic_square
-    from numpy.testing import suppress_warnings
-    from scipy.optimize._remove_redundancy import _remove_redundancy, _remove_redundancy_dense, _remove_redundancy_sparse
-    from scipy.optimize._linprog_util import _presolve, _clean_inputs, _LPProblem
-    from scipy.sparse import csc_matrix, csr_matrix, issparse
-    import numpy as np
-    import os
-except ImportError:
```

Comment:
I'm almost certain we do this intentionally in benchmarks to allow for them to run smoothly over a very wide range of commits. You'll find it in many different places so that if a feature is missing the benchmarks may continue past that point in history.

@pv can correct me if I'm wrong

Reply:
So while it may make debugging a little harder when running in a target manner, I think I'm probably -0.5 here at best.

Reply:
I don't know why `os` and `numpy` are being guarded though, so some things probably don't need to be. Just the SciPy features.